data/state/interface.go

```diff
@@ -22,8 +22,8 @@ const HashLength = 32
 // PubkeyConverter can convert public key bytes to/from a human readable form
 type PubkeyConverter interface {
 	Len() int
-	Bytes(humanReadable string) ([]byte, error)
-	String(pkBytes []byte) (string, error)
+	Decode(humanReadable string) ([]byte, error)
+	Encode(pkBytes []byte) string
```

Comment:
Can all pkBytes be encoded? Is there no need for the error?

Reply:
as tested, any bytes can be encoded. The errors were mostly returned due to a bad config, which are signaled as warn log messages (programming error)
Without returning the error, the code is more lightweight

Reply:
ok